docs/config-spec.md

```diff
@@ -17,73 +17,68 @@ For example:
 # Metadata
 #####################
 
-# TBD
-# Unique identifier for this observability pack
-# uuid: guid, required
+# A unique Observability Pack identifier
+uuid: guid, required
+
+# The name of the pack displayed everywhere the pack is referenced
+title: string, optional
 
 # Name acts as a unique identifier for this observability pack
 # Formatted as hyphenated lowercase
 # Example: apache, golden-signals-for-web-servers
 name: string, required
 
-# Displayed on the pack overview/details page formatted into rich text.
-# Standardized to a template for the New Relic packs.
+# A long form description for this Observability Pack.
 description: string, required
 
-# TBD
-# url slug for linking on public website
-# recommended format: make this a lowercase and hyphenated version
-# of the `name` field
-# slug: string, required
+# Displayed in search results and recommendations. Summarizes a packs functionality.
+# A short form description for this Observability Pack.
+summary: string, required
+
+# URL to link to the location of this Observability Pack in the open source repo
+packUrl: string, optional
```

Comment:
This is purely a field in the GraphQL API, it will not be present in the pack configuration.

Reply:
gotcha.. so a user won't be contributing this value. I can remove it.

Reply:
actually I'd like to keep this documented for now, until we get our guides in place and the json schema updated. but i'll add a comment clarifying

Reply:
IMO this doesn't belong here. A user will not enter this and it's misleading to say it's a part of the config spec. It also shouldn't be a part of the contributing guide - user's shouldn't be submitting this as valid pack input, it will just be ignored.

Reply:
I removed it.

Reply:
@jpvajda Looks like this got re-added!

Reply:
💇  it's gone now.